procgov-tests/ProcessGovernorUnitTests.cs

```diff
@@ -113,5 +114,16 @@ public void PrepareDebuggerCommandStringTest()
                 File.Delete(envFilePath);
             }
         }
+
+        [Test]
+        public void StartProcessTest()
```

Comment:
It would be better if this test could assert on the actual arguments that are passed to `CreateProcess`. However this would require refactoring of the code under test.